drivers/ads101x/ads101x_saul.c

```diff
@@ -1,6 +1,6 @@
 /*
  * Copyright (C) 2017 OTA keys S.A.
- *               2018 Matthew Blue <matthew.blue.neuro@gmail.com>
+ *               2018 Acutam Automation, LLC
```

Comment:
unrelated, and unclear why? Here and other files

Reply:
I have transferred my copyrights on RIOT code from myself (Matthew Blue) to my company (Acutam Automation, LLC). I figured it was a minor enough change that it did not require a separate PR, or detailed explanation.

Reply:
ah okay, sorry was confused by GitHub username vs author name, never mind.